### PR TITLE
feat: Make the releaseCount configurable

### PR DESCRIPTION
**Pull request by natioskar**
Hey! Thank you for this action.

Resolves https://github.com/TriPSs/conventional-changelog-action/issues/7

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ This action will bump version, tag commit and generate a changelog with conventi
 - **Optional** `preset`: Preset that is used from conventional commits. Default `angular`.
 - **Optional** `tag-prefix`: Prefix for the git tags. Default `v`.
 - **Optional** `output-file`: File to output the changelog to. Default `CHANGELOG.md`.
+- **Optional** `changelog-release-count`: FNumber of releases to preserve in changelog. Default `5`, use `0` to regenerate all.
```
Comment (TriPSs) [inline README.md]: `FNumber` typo?

Comment (natioskar) [reply README.md]: Fixed

**Comment by TriPSs** [comment]
Thanks for the PR!

**Comment by natioskar** [comment]
np :) are you planing on creating release and tag for the new version?

**Comment by TriPSs** [comment]
@natioskar Released in `v1.3.0` :)

**Pull request merged**